src/tiles/tiles.ts

```diff
@@ -231,22 +231,22 @@ export function tileToBoundingBox(tile: Tile, referenceTileGrid: TileGrid = TILE
   const width = tileEffectiveWidth(tile.z, referenceTileGrid) * metatile;
   const height = tileEffectiveHeight(tile.z, referenceTileGrid) * metatile;
 
-  const boundingBox: BoundingBox = new BoundingBox(
+  const boundingBox: BoundingBox = new BoundingBox([
```

Comment:
Why do these calculations if `clamp = true`? Please move the condition to be first.

Reply:
it can clamp the bounding box after it's values are calculated

Reply:
maybe the clamp functionality can be extracted

Reply:
Ok, did you make these changes?